src/standalone_async/hello_async.cpp

```diff
@@ -105,7 +105,7 @@ struct AsyncHelloWorker : Nan::AsyncWorker {
         callback->Call(argc, static_cast<v8::Local<v8::Value>*>(argv));
     }
 
-    std::string result_;
+    std::string result_{};
```

Comment:
Note: this `{}` was to appease `-Weffc++` per https://github.com/mapbox/cpp/issues/54